api/src/main/java/org/pmiops/workbench/access/AccessTierServiceImpl.java

```diff
@@ -205,6 +201,20 @@ private void removeFromAuthDomainIdempotent(DbUser dbUser, DbAccessTier accessTi
     }
   }
 
+  @Override
+  public List<DbUser> getAllRegisteredTierUsers() {
+    return userAccessTierDao.getAllByAccessTier(getRegisteredTierOrThrow()).stream()
+        .filter(uat -> uat.getTierAccessStatusEnum() == TierAccessStatus.ENABLED)
+        .map(DbUserAccessTier::getUser)
+        .collect(Collectors.toList());
+  }
+
+  private DbAccessTier getRegisteredTierOrThrow() {
+    return accessTierDao
+        .findOneByShortName(REGISTERED_TIER_SHORT_NAME)
+        .orElseThrow(() -> new ServerErrorException("Cannot find Registered Tier in database."));
```

Comment:
I don't have enough context here: Is 500 the correct exception we should throw?  Should it be 404?

Reply:
yes.  If the app has no Registered Tier, something is seriously broken.